src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -24,6 +24,7 @@ const StartDate = props => {
       }}
       placeholder="Start Date"
       invalid={!props.canEdit}
+      max={props.userProfile.endDate ? moment(props.userProfile.endDate).format('YYYY-MM-DD') : ''}
```

Comment:
Just to double check, the place holder is Start Date and the endDate property is being used here, is this correct?

Reply:
Yes this is correct. Please refer to the PR description to understand context.